src/js/tracker.js

```diff
@@ -1112,21 +1095,20 @@
 		function getOptimizelyXSummary() {
 			var state = getOptimizelyXData('state');
 			var experiment_ids = state.getActiveExperimentIds();
-			var experiments = getOptimizelyXData('data', 'experiments');
 			var visitor = getOptimizelyXData('visitor');
 
 			return map(experiment_ids, function(activeExperiment) {
 				var variation = state.getVariationMap()[activeExperiment];
```

Comment:
Maybe you could pull `state.getVariationMap()` up outside of the loop, so you don't have to re-call it for each activeExperiment in the array.